.github/workflows/workflow_test.yml

```diff
@@ -4,14 +4,16 @@ name: Galaxy Workflow Tests for push and PR
 on: [push, pull_request]
 env:
   GITHUB_USER: iwc-workflows-bot
+  GALAXY_FORK: galaxyproject
+  GALAXY_BRANCH: release_24.0
 jobs:
   setup:
     name: Setup cache and determine changed repositories
     uses: ./.github/workflows/setup.yml
     with:
       python-version-list: "[\"3.11\"]"
-      galaxy-fork: galaxyproject
-      galaxy-branch: release_23.2
+      galaxy-fork: ${{ env.GALAXY_FORK }}
```

Comment:
Have you tested that this really works ? I am not sure in what order env vars and job matrices are being evaluated.

Reply:
Nope. Is there an easy way, e.g. bump a workflow?

Reply:
`s/23.2/24.0/g`